chapter-2-drivers.md

```diff
@@ -62,6 +62,8 @@ public IInOutDriver<bool, bool> Driver
 }
 ```
 
+A method `OnInitialize` should exist.
```

Comment:
Not sure if we can just take this for granted as we already talk explain this a few lines above,